assets/optional/multus/07-daemonset-dhcp.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       containers:
       - name: dhcp-daemon
         image: containernetworking-plugins-microshift
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Do we need to update the rebase script as well?

Reply:
AFAIK we dont  rebase multus from upstream,
this is the reference:
https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/assets.yaml#L257

@pmtk , correct me if am wrong..